pkg/api/v1/defaults.go

```diff
@@ -228,6 +228,10 @@ func SetDefaults_PersistentVolumeClaim(obj *v1.PersistentVolumeClaim) {
 	if obj.Status.Phase == "" {
 		obj.Status.Phase = v1.ClaimPending
 	}
+	if obj.Spec.VolumeType == nil {
+		obj.Spec.VolumeType = new(v1.PersistentVolumeType)
+		*obj.Spec.VolumeType = v1.PersistentVolumeFile
```

Comment:
you need a corresponding default in the `pkg/api/fuzzer.go` function that sets up a PVC object to test round tripping